resources/zib/zib-ComfortScale.xml

```diff
@@ -0,0 +1,353 @@
+  <title value="zib ComfortScale" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="email" />
+      <value value="info@nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="The Comfort Behaviour scale /Comfortneo scale is an assessment tool, that is used to assess the pain behaviour of neonates (new born).&#xD;&#xA;The Comfort Behaviour scale has 7 observation points that that need to be addressed. For each on observation is value is assigned from a 5-point scale, the final score indicates whether the observed child suffers pain and 'how much'. The score is combined with an NRS score that is completed by the nurses on basis of clinical experience." />
+  <purpose value="This Observation resource represents the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) ComfortScale-v1.1(2020EN)](https://zibs.nl/wiki/ComfortScale-v1.1(2020EN))." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-comfortscale-v1.1-2020EN" />
+    <uri value="https://zibs.nl/wiki/ComfortScale-v1.1(2020EN)" />
+    <name value="zib ComfortScale-v1.1(2020EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="true" />
+  <type value="Observation" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Observation" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Observation">
+      <path value="Observation" />
+      <short value="ComfortScale" />
+      <alias value="ComfortScore" />
+      <mapping>
+        <identity value="zib-comfortscale-v1.1-2020EN" />
+        <map value="NL-CM:12.12.1" />
+        <comment value="ComfortScale" />
+      </mapping>
+    </element>
+    <element id="Observation.code">
+      <path value="Observation.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://snomed.info/sct" />
+          <code value="108301000146109" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Observation.effective[x]">
+      <path value="Observation.effective[x]" />
+      <slicing>
+        <discriminator>
+          <type value="type" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.effective[x]:effectiveDateTime">
+      <path value="Observation.effectiveDateTime" />
+      <sliceName value="effectiveDateTime" />
```

Comment:
Dit moet `Observation.effective[x]` zijn. Een resave in Forge lost dit op.

Reply:
Resave fix